### PR TITLE
Put report builder behind its own flag

### PR DESCRIPTION
**Pull request by NoahCarnahan**
@czue cc @dannyroberts

### CODE REVIEW

**Comment by dannyroberts** [comment]
(not mergeable)

**Comment by NoahCarnahan** [comment]
merge conflict fixed

File: corehq/apps/prelogin [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 4172d946d67cd283b6043e304ff7c32286a377b4
```
Comment (dannyroberts) [inline corehq/apps/prelogin]: Did you mean to update this submodule? Doesn't seem like that belongs in this PR

Comment (NoahCarnahan) [inline corehq/apps/prelogin]: gah, I did not, thanks.

Comment (dannyroberts) [inline corehq/apps/prelogin]: My guess is when resolving the merge conflict you by mistake added this as well?

**Comment by NoahCarnahan** [comment]
Fixed for real now

**Comment by czue** [comment]
looks good - thanks!

**Pull request merged**